packages/components/src/components/chat/MarkdownCodeSnippet.vue

```diff
@@ -94,7 +94,7 @@ export default Vue.extend({
     // project directory, we'll just return the full path, what ever it may be.
     shortPath(): string | undefined {
       // If we're using a URI and it's not a file, skip this.
-      if (this.uriComponents && this.uriComponents?.scheme !== 'file') return undefined;
+      if (!this.decodedLocation) return undefined;
```

Comment:
The comment above is out of date now, I believe.

Reply:
https://github.com/getappmap/appmap-js/pull/2278/commits/c763dd8e922175c2fbb737aed453d04ad24bc2b7